src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -958,19 +958,6 @@ public function getModel()
     }
 
     /**
-     * Get a fresh instance of a model instance being queried.
-     *
-     * @param  array  $attributes
-     * @return \Illuminate\Database\Eloquent\Model
-     */
-    public function newModel(array $attributes = [])
```

Comment:
@GrahamCampbell Well I added this method yesterday https://github.com/laravel/framework/pull/12328 since I didn't know I can use $model->newInstance(). I think we can still safely remove it - it's only been a day.